platform/admin/core/admin-core-configpolicy/src/main/java/org/codice/ddf/admin/config/policy/AdminConfigPolicy.java

```diff
@@ -0,0 +1,172 @@
+            //If there are multiple features in the permission and one is not authorized, the user is not authorized to see any of the features in the group
+            for (String matchPermissionValue : ((KeyValuePermission) permission).getValues()) {
+                List<KeyValueCollectionPermission> matchOneAttributes = policyPermissions.get(
+                        matchPermissionValue);
+
+                //If null, there is no configuration with this attribute in the policy, the feature or service is white listed
+                if (matchOneAttributes != null) {
+                    boolean matchedOneAttribute = false;
+
+                    for (KeyValueCollectionPermission attributePermissions : matchOneAttributes) {
+                        if (subjectAllCollection.implies(attributePermissions)) {
+                            matchedOneAttribute = true;
+                            break;
+                        }
+                    }
+
+                    if (!matchedOneAttribute) {
+                        isPermitted = false;
+                        break;
+                    }
+                }
+            }
+
+            if (isPermitted) {
+                newMatchOneCollectionPermissions.remove(permission);
+            }
+
+        }
+        return new KeyValueCollectionPermission(matchOneCollection.getAction(),
+                newMatchOneCollectionPermissions.stream()
+                        .toArray(KeyValuePermission[]::new));
+    }
+
+    public void setFeaturePolicies(List<String> featurePolicies) {
+        featurePolicyPermissions.clear();
+        featurePolicyPermissions.putAll(parsePermissions(featurePolicies));
+    }
+
+    public void setServicePolicies(List<String> servicePolicies) {
+        servicePolicyPermissions.clear();
+        servicePolicyPermissions.putAll(parsePermissions(servicePolicies));
+    }
+
+    public Map<String, List<KeyValueCollectionPermission>> parsePermissions(List<String> policies) {
+
+        Map<String, List<KeyValueCollectionPermission>> newPolicyPermissions = new HashMap<>();
+
+        for (String policy : policies) {
+
+            if (StringUtils.isEmpty(policy)) {
+                continue;
+            }
+
+            //Example input: featureName="attributeName=attributeValue","attributeName2=attributeValue2"
+            String[] policyTrimmed = policy.replaceAll("\\s+", "")
+                    .split("=", 2);
+            String permissionName = policyTrimmed[0];
+            String policyAttributes = policyTrimmed[1];
+
+            List<KeyValueCollectionPermission> permissionAttributeMap = new ArrayList<>();
+
+            for (String policyAttribute : policyAttributes.split(",")) {
+                policyAttribute = policyAttribute.replace("\"", "");
+                policyAttribute = policyAttribute.replaceAll("\\s+", "");
+                String attributeName = policyAttribute.split("=")[0];
```

Comment:
I would probably split just once and save it to a local variable that you then get elements `[0]` and `[1]` from. The split is a moderately expensive operation.